src/rmats2sashimiplot/rmats2sashimiplot.py

```diff
@@ -63,37 +72,39 @@ def checkout(parser, options):
     """
     # bam files and sam files are alternative, the same for the case of events_file and coordinate
     # events_file should be provided together with event_type
-    if (options.s1 is None and options.b1 is None) or (options.s2 is None and options.b2 is None):
-        parser.error("Not enough arguments! Please provide sam or bam files.")
+    if options.s1 is None and options.b1 is None:
+        parser.error("Not enough arguments! Please provide either --s1 or --b1")
+    if (((options.s1 is not None or options.s2 is not None)
+         and (options.b1 is not None or options.b2 is not None))):
+        parser.error("Specify either sam files or bam files not both")
     if (options.events_file is None or options.event_type is None) and options.coordinate is None:
         parser.error("Not enough arguments! Please provide "
                      "1) coordinates with gff3 files. or "
                      "2) events files together with events type.")
 
```

Comment:
`if options.s2 is not None:`
`elif options.b2 is not None`
decrease indentation